packages/platform/types/bookings/2024-08-13/inputs/create-booking.input.ts

```diff
@@ -28,13 +55,18 @@ class Attendee {
   @IsString()
   name!: string;
 
-  @ApiProperty({
+  @ApiPropertyOptional({
     type: String,
     description: "The email of the attendee.",
     example: "john.doe@example.com",
+    default: "", // Kept the default empty string
+  })
+  @IsOptional()
+  @Validate((value: string) => !value || isEmail(value), {
+    message: "Invalid email format",
   })
-  @IsEmail()
-  email!: string;
+  @RequireEmailOrPhone()
+  email = ""; // Default to empty string
```

Comment:
I would not default to empty email but instead make it optional:
```
email?: string;
```

Reply:
Actually handleNewBooking expects email as string because right now in DB the expected type is string. for phone based booking we convert phone number to a placeholder email :- +91 98765432100 --> 919876543210@sms.cal.com